pkgs/servers/monitoring/prometheus/dmarc-metrics-exporter/default.nix

```diff
@@ -1,28 +1,33 @@
 { lib
 , python3
+, fetchFromGitHub
 }:
 
 python3.pkgs.buildPythonApplication rec {
   pname = "dmarc-metrics-exporter";
-  version = "0.6.1";
+  version = "0.8.0";
 
   disabled = python3.pythonOlder "3.7";
 
   format = "pyproject";
 
-  src = python3.pkgs.fetchPypi {
-    inherit pname version;
-    hash = "sha256-VYmSHDde3zLq7NcsX7xp1JYe6x6RKFEravpakIzW390=";
+  src = fetchFromGitHub {
+    owner = "jgosmann";
+    repo = "dmarc-metrics-exporter";
+    rev = "v${version}";
+    hash = "sha256-uZCPEUoWNrcSz5qV24eFdZOOxKkoz+Qn6HpELBwOK2I=";
   };
 
+  pythonRelaxDeps = true;
+
   postPatch = ''
     substituteInPlace pyproject.toml \
-      --replace 'uvicorn = {extras = ["standard"], version = "^0.15.0"}' 'uvicorn = {version = "^0.15.0"}' \
-      --replace '"^' '">='
+      --replace 'uvicorn = {extras = ["standard"],' 'uvicorn = {' \
```

Comment:
shouldn't be necessary once https://github.com/NixOS/nixpkgs/pull/206834 is merged

Reply:
You can remove `postPatch` now.